src/components/BlogPost/PostNavigation.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'gatsby';
+
+import { FaCaretLeft, FaCaretRight } from 'react-icons/fa';
+
+const PostNavigation = ({ prev, next }) => (
+  <div className="flex items-center h-24">
+    { prev
+          && (
+          <>
+            <div className="flex-2 text-center px-4 py-2 m-2">
```

Comment:
I think it would be better to use one Link to wrap both the icon and text, e.g.

```jsx
<Link ...>
    <div className="...">
        <FaCaret ... className="inline mr-4" />
        Previous
    </div>
</Link>
```

this way we could also get rid of the duplicate code with the link component 😊

Reply:
doing this makes takes away the flexibility of each component. As it is right now, all four components are properly sized and spaced. Putting Link as child of the main flex div will not let it remain flexible.

Reply:
Hm but I would consider the caret to be part of the button, so I would expect it to be always placed relative to the `Previous` text. I think a shared button component makes sense here.
You're right, it does take away some flexibility here, but I think I'd prefer not having too much duplicate here.